src/cryojax/inference/__init__.py

```diff
@@ -1,2 +1,2 @@
 from . import distributions as distributions
-from . import parameters as parameters
+from . import transforms as transforms
```

Comment:
why can't you just do `from . import transforms`? Why do you need the `as`?

Reply:
this is an annoying detail in `__init__.py`s where if you do not have the `as`, you get a message like `'transforms' is not accessed` from linters, like Pylance. I'm following the pattern of the equinox ecosystems `__init__.py`s